metamorphic/ops.go

```diff
@@ -102,6 +102,16 @@ type checkpointOp struct {
 }
 
 func (o *checkpointOp) run(t *test, h historyRecorder) {
+	// TODO(josh): db.Checkpoint does not work with shared storage yet.
+	// It would be better to filter out ahead of calling run on the op,
+	// by setting the weight that generator.go uses to zero, or similar.
+	// But IIUC the ops are shared for ALL the metamorphic test runs, so
+	// not sure how to do that easily:
```

Comment:
This works for now. I was thinking that we just create a new `TestMetaShared` test that enables shared storage options and disables the checkpoint op. But that duplicates test churn without that much of an additional win, so faking a successful checkpoint in the record is good enough.

Reply:
Thanks!